base/src/main/java/org/aya/core/def/PrimDef.java

```diff
@@ -217,7 +217,7 @@ public Factory() {
           init.stringConcat,
           init.intervalType,
           init.partialType,
-          init.coerce,
+          init.coe,
```

Comment:
Tbh I personally prefer longer names in the compiler...

Reply:
Okay, but for `coerce` I guess it's fine, because I decide to name `coeInv` as `eoc` so `coerce` looks wired as `ecreoe`